### DISCUSSION TITLE
[Question]:  Bounding Boxes not lining up

### DISCUSSION THREAD

**Discussion by Frikkie50** [blakeblackshear/frigate#9988] [Ask A Question]
### What is your question:

Just a question. Is there a secret to the bounding boxes to line up with the recording? I have tried setting Frigate up with a sub_stream + Annotation offset and it seems the timing keeps changing. I tried using the main stream for detect & record but it still doesnt line up. 

Any other ideas?

**Comment by NickM-27** [comment]
there will always be some latency between the streams and some cameras are more consistent than others
